src/org/omegat/core/machinetranslators/MTHUBTranslate.java

```diff
@@ -0,0 +1,412 @@
+    }
+
+    public void setTranslation(String translation) {
+        this.translation = translation;
+    }
+}
+
+/**
+ * A class for keep the response debug from MT-Hub.
+*/
+class MTHUBTranslateResponseDataDebug {
+    public int supplierId;
+    public String engineName;
+
+    public int getSupplierId() {
+        return supplierId;
+    }
+
+    public void setSupplierId(int supplierId) {
+        this.supplierId = supplierId;
+    }
+
+    public String getEngineName() {
+        return engineName;
+    }
+
+    public void setEngineName(String engineName) {
+        this.engineName = engineName;
+    }
+}
+
+/**
+ * A class for keep the response data from MT-Hub.
+*/
+class MTHUBTranslateResponseData {
+    public List<List<MTHUBTranslateResponseDataSegments>> segments;
+    public List<MTHUBTranslateResponseDataDebug> debug;
+
+    public List<List<MTHUBTranslateResponseDataSegments>> getSegments() {
+        return segments;
+    }
+
+    public void setSegments(List<List<MTHUBTranslateResponseDataSegments>>
+                            segments) {
+        this.segments = segments;
+    }
+
+    public List<MTHUBTranslateResponseDataDebug> getDebug() {
+        return debug;
+    }
+
+    public void setDebug(List<MTHUBTranslateResponseDataDebug> debug) {
+        this.debug = debug;
+    }
+}
+
+/**
+ * A class for keep the response error from MT-Hub.
+*/
+class MTHUBTranslateResponseError {
+    public int statusCode;
+    public int code;
+    public int timestamp;
+    public String message;
+
+    public int getStatusCode() {
+        return statusCode;
+    }
+
+    public void setStatusCode(int statusCode) {
+        this.statusCode = statusCode;
+    }
+
+    public int getCode() {
+        return code;
+    }
+
+    public void setCode(int code) {
+        this.code = code;
+    }
+
+    public int getTimestamp() {
+        return timestamp;
+    }
+
+    public void setTimestamp(int timestamp) {
+        this.timestamp = timestamp;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+}
+
+/**
+ * A class for keep the response from MT-Hub.
+*/
+class MTHUBTranslateResponse {
+    public boolean success;
+    public List<String> error;
+    public List<MTHUBTranslateResponseData> data;
+
+    public boolean isSuccess() {
+        return success;
+    }
+
+    public void setSuccess(boolean success) {
+        this.success = success;
+    }
+
+    public List<String> getError() {
+        return error;
+    }
+
+    public void setError(List<String> error) {
+        this.error = error;
+    }
+
+    public List<MTHUBTranslateResponseData> getData() {
+        return data;
+    }
+
+    public void setData(List<MTHUBTranslateResponseData> data) {
+        this.data = data;
+    }
+}
+
+public class MTHUBTranslate extends BaseTranslate {
+    private static final String PROPERTY_API_KEY = "MTHUB.api.key";
```

Comment:
I think this needs to be all lowercase to match what you're setting on the command line in `omegat.kaptn`.